lib/registration.pm

```diff
@@ -119,12 +135,20 @@ sub fill_in_registration_data {
                     save_screenshot;
                 }
             }
-            send_key 'alt-n';    # next
-            while (check_screen('import-untrusted-gpg-key', 20)) {
-                record_soft_failure 'untrusted gpg key';
-                send_key 'alt-t', 2;
+            send_key 'alt-n';
+            # start addons/modules registration, it needs longer time if select multiple or all addons/modules
+            while (assert_screen(['import-untrusted-gpg-key', 'yast_scc-pkgtoinstall', 'inst-addon'], 120)) {
```

Comment:
I think a

```perl
while (1) {
    assert_screen(…);
    …
    if (…) {
    …
    last;
    }
}
```

is better but maybe even

```perl
until (match_has_tag('inst-addon') || match_has_tag('yast_scc-pkgtoinstall')) {
}
```
or something like that but because I don't know what exactly should be there … nevermind :-)